src/charactersheet/viewmodels/common/notes/index.js

```diff
@@ -54,11 +54,21 @@ export function NotesViewModel() {
         self.selectNote(note);
     };
 
+    self.noteToSelect = function(note) {
+        var previousNote = self.notes().indexOf(note) -1;
```

Comment:
Another minor nitpick, can you put a space between the `-` and the `1`?

Reply:
Sure